src/features/devices/hooks/target-hostapp.ts

```diff
@@ -0,0 +1,296 @@
+import {
```

Comment:
Does this make sense to move into a `hostapp` feature instead of the fairly generic `devices` feature?

Reply:
yeah, that's fine with me